src/test/java/com/facebook/openwifirrm/services/MockOWSecService.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.openwifirrm.services;
+
+import java.util.Map;
+import java.util.HashMap;
+import java.time.Instant;
+
+import com.facebook.openwifirrm.ucentral.gw.models.TokenValidationResult;
+import com.facebook.openwifirrm.ucentral.gw.models.UserInfo;
+import com.facebook.openwifirrm.ucentral.gw.models.WebTokenResult;
+
+import com.google.gson.Gson;
+import spark.Service;
+import spark.Request;
+import spark.Response;
+import spark.Route;
+
+public class MockOWSecService {
```

Comment:
prefer if you add a class-level comment for future reference, or just put the URL:
```
/**
 * @see <a href="https://github.com/Telecominfraproject/wlan-cloud-ucentralsec">owsec</a>
 */
```